Presentation.Web/app/components/org/user/org-user.controller.ts

```diff
@@ -307,6 +312,17 @@
                             }
                             return `<span data-ng-model="dataItem.OrganizationUnitRights" value="rights.Role.Name" ng-repeat="rights in dataItem.OrganizationUnitRights"> {{rights.Role.Name}}{{$last ? '' : ', '}}</span>`;
                         },
+                        excelTemplate: (dataItem) => {
+                            var rightNames = "";
+                            dataItem.OrganizationUnitRights.forEach((right, index) => {
+                                rightNames += right.Role.Name;
```

Comment:
why not use join in stead?
`dataItem.OrganizationUnitRights.map(right=>right.Role.Name).join(", ");`

That takes care of the "last item" issue